sdk/src/main/java/com/hedera/hashgraph/sdk/EntityIdHelper.java

```diff
@@ -40,7 +42,7 @@
 /**
  * Utility class used internally by the sdk.
  */
-class EntityIdHelper {
+public class EntityIdHelper {
```

Comment:
note:
Made this class public in order to access it from the tests, changed most public methods to package private